.github/workflows/tests.yml

```
@@ -0,0 +1,40 @@
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install pytest pytest-cov pytest-mock
+        pip install -e .
+    - name: Run tests
+      run: |
+        python -m pytest --cov=src/tag_version tests/
+    - name: Generate coverage report
+      run: |
+        python -m pytest --cov=src/tag_version --cov-report=xml tests/
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)